product_cost_usd/demo/product_pricelist_demo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="pricelist_15_usd" model="product.pricelist">
+            <field name="name">Pricelist 15% USD</field>
+            <field name="currency_id" ref="base.USD" />
+            <field name="item_ids" eval="[
+                (5, 0, 0),
```

Comment:
I consider it's better to create pricelist items below, because they need to be preserved. This is not as in views, that they can be removed and re-created without any issue.

Of course, this will only occur if running `-i` again or if this file is reloaded, but better following good practices